### DISCUSSION TITLE
Any help needed?

### DISCUSSION THREAD

**Discussion by nikolaikabanenkov** [OutlineFoundation/outline-sdk#314] [General]
Hi,

I’m a Go developer at a FinTech company, and I’ve been using an app based on Outline SDK for a while, and I love it.

That's why I would really like to contribute to the project. Are there any areas where you’re looking for help, or any guidance on how I can get started?

Sincerely,
Nikolai

**Comment by fortuna** [comment]
Thanks for offering help!

To respond to your request, I started creating feature request under the "help wanted" label: https://github.com/Jigsaw-Code/outline-sdk/labels/help%20wanted

Please let us know if you can help with any of those. And we will keep thinking of other ways people can help us.

**Reply by nikolaikabanenkov** [reply]
@fortuna , thanks for the reply!

I think that https://github.com/Jigsaw-Code/outline-sdk/issues/319 is a good one to begin with.
